google-cloud-spanner/lib/google/cloud/spanner/service.rb

```diff
@@ -144,12 +146,12 @@ def create_instance instance_id, name: nil, config: nil, nodes: nil,
           instances.create_instance request, opts
         end
 
-        def update_instance instance, call_options: nil
+        def update_instance instance, field_mask: nil, call_options: nil
```

Comment:
This seems to be a breaking change. If the user doesn't provide a `field_mask` in the new version, will `update_instance` updates `display_name`, `node_count`, and `labels`?

Reply:
Consider to use `%w[display_name node_count labels]` as a default value of `field_mask`.

Reply:
ok, I will make necessary changes.

Reply:
Updated logic. Let me know your suggestions.

Reply:
Looks good to me.